### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Sibghat-Ullah-26**

### CODE REVIEW

File: const.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+const roleEnum = ['admin', 'superadmin', 'manager', 'user'];
```
Comment (hassan493) [inline const.js]: Camel Case(preferred) or underscore case.
But it should be unified all through the app.

File: controllers/attendees.controller.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+async function getAttendees(req, res) {
+  let attendees = await attendeesService.getAttendees();
+  res.send(attendees);
+}
+
+async function getAttendeesById(req, res) {
+  let { id } = req.params;
+  id = Number(id);
+  let attendees = await attendeesService.getAttendeesById(id);
```
Comment (hassan493) [inline controllers/attendees.controller.js]: it's a single attendee , not attendees.

File: controllers/attendees.controller.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  if (!attendees) {
+    return res.send({ message: 'Attendee does not exist' });
+  } else res.send(attendees);
+}
+
+async function updateAttendees(req, res) {
+  let { id } = req.params;
+  id = Number(id);
+  let reqObject = req.body;
+  const { error } = attendeesValidator.attendeesSchema.validate({
```
Comment (hassan493) [inline controllers/attendees.controller.js]: extra spreading

File: controllers/attendees.controller.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    ...reqObject,
+  });
+  if (error !== undefined) {
+    return res.send({ message: 'Invalid Body', error });
+  }
+  let updateAttendees = await attendeesService.updateAttendee({
+    where: { id },
+    data: {
```
Comment (hassan493) [inline controllers/attendees.controller.js]: if there is required addition of any controller added properties , follow the spread syntax else pass the object directly.

File: controllers/attendees.controller.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+      ...reqObject,
+    },
+  });
+  res.send(updateAttendees);
+}
+
+async function deleteAttendees(req, res) {
+  let { id } = req.params;
+  id = Number(id);
+  let deleteAttendees = await attendeesService.deleteAttendee(id);
```
Comment (hassan493) [inline controllers/attendees.controller.js]: function name and variable name wrong according to action performed. It's against intentional rule of naming.

File: controllers/exceptionSlots.controller.js [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+module.exports = {
+  createExceptionSlots,
+  getExceptionSlots,
+  getExceptionSlotsById,
+  updateExceptionSlots,
+  deleteExceptionSlots,
+};
+
+async function createExceptionSlots(req, res) {
```
Comment (hassan493) [inline controllers/exceptionSlots.controller.js]: single slot creation. (intentional rule)

File: controllers/exceptionSlots.controller.js [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  let updateexceptionSlot = await exceptionSlotsService.updateExceptionSlots({
+    where: { id },
+    data: {
+      ...reqObject,
+    },
+  });
+  res.send(updateexceptionSlot);
+}
+
+async function deleteExceptionSlots(req, res) {
```
Comment (hassan493) [inline controllers/exceptionSlots.controller.js]: intentional naming

File: routes/index.js [hunk 8]
```diff
@@ -1,7 +1,21 @@
...
+const scheduledSlotsRoutes = require("./scheduleSlot")
+const attendeesRoutes = require("./attendees")
 const router = express.Router();
 
 module.exports = router;
 
 router.use("/users", usersRoutes);
+router.use("/meetings", meetingsRoutes);
+router.use("/exceptionslots", exceptionSlotRoutes);
+router.use("/feedback", feedbackRoutes);
```
Comment (hassan493) [inline routes/index.js]: singular , keep it plural

**Review by hassan493 — CHANGES_REQUESTED** [review]
Change what commented

**Comment by Sibghat-Ullah-26** [comment]
sure.

**Pull request closed without merging**